src/test/codegen/swap-small-types.rs

```diff
@@ -18,6 +18,22 @@ pub fn swap_rgb48(x: &mut RGB48, y: &mut RGB48) {
     swap(x, y)
 }
 
+// CHECK-LABEL: @swap_vecs
+#[no_mangle]
+pub fn swap_vecs(x: &mut Vec<u32>, y: &mut Vec<u32>){
+// CHECK-NOT: alloca
+// CHECK: ret void
+    swap(x, y)
+}
+
+// CHECK-LABEL: @swap_slices
+#[no_mangle]
+pub fn swap_slices<'a>(x: &mut &'a[u32], y: &mut &'a[u32]){
```

Comment:
nit: rustfmt would rather this be
```suggestion
pub fn swap_slices<'a>(x: &mut &'a [u32], y: &mut &'a [u32]) {
```

Reply:
Run `rustfmt`.
Seems that it was never run on that file before.